gcloud/bigquery/dataset.py

```diff
@@ -64,6 +64,8 @@ def __init__(self, name, client, access_grants=()):
         self.name = name
         self._client = client
         self._properties = {}
+        self._access_grants = None
+        # Let the @property do validation.
```

Comment:
Yeah I know. I debated splitting out the validation behavior into a separate method.

The complaint was that a value on the instance was initiated somewhere other than the constructor. I forgot your smart solution: setting `None` on the class.

Let me circle back and fix this?